### PR TITLE
Remove getting_started message, change message copy for successful_submission_online_intake message

### PR DESCRIPTION
**Pull request by embarnard**

### CODE REVIEW

**Comment by embarnard** [comment]
Update the `successful_submission_online_intake` message. This is the new copy below.
Screen Shot 2023-01-24 at 12 59 05 PM

And this is the old copy:
Screen Shot 2023-01-24 at 1 00 16 PM

This story also removes the getting_started message. Originally we were going to change/add back in the in_progress message but now its going to be a different message and included in another ticket.

File: config/locales/en.yml [hunk 1]
```diff
@@ -1492,23 +1468,21 @@ en:
...
+          Your tax team at GetYourRefund
+        subject: Welcome to GetYourRefund/ Client ID
       sms: |
-        Hello <<Client.PreferredName>>,
-        Your tax information has been successfully submitted! Your tax specialist will review your information and reach out in 3-5 business days for a phone review of your situation before preparing your taxes.
-        Check your progress and add additional documents here: <<Client.LoginLink>>
-        If you have any questions, please reply to this message.
-        We’re here to help!
-        Your tax team at GetYourRefund.org
+        Hello <<Client.PreferredName>>, Your tax information was successfully submitted to GetYourRefund**.** IMPORTANT**:** Your Client ID is <<Client.ClientId>>. Please keep track of this number and do not delete this message.
```
Comment (alaroia-cfa) [inline config/locales/en.yml:1483]: In terms of English punctuation/spacing, I think this should be `**IMPORTANT**` not `** IMPORTANT**`.

However, are you sure program/product wants these asterisks here? I don't see them in the Notion doc.

**Review by alaroia-cfa — CHANGES_REQUESTED** [review]
See comments for two blocking requests to confirm copy.

See other comments for some non-blocking questions/comments.

Happy to chat more!

File: config/locales/en.yml [hunk 2]
```diff
@@ -1492,23 +1468,21 @@ en:
...
           Check your progress and add additional documents here: <<Client.LoginLink>>
 
           If you have any questions, please reply to this message.
 
           We’re here to help!
-
-          Your tax team at GetYourRefund.org
-        subject: Your tax information has been successfully submitted!
+          Your tax team at GetYourRefund
+        subject: Welcome to GetYourRefund/ Client ID
```
[the remark this answers is not in the document]
Comment (embarnard) [reply config/locales/en.yml:1481]: @alaroia-cfa I didn't mean to have that difference `/` vs `:` so I'll change it to the colon. Thanks for the thorough review!

**Pull request merged**